app/src/main/res/layout/layout_switch_drive.xml

```diff
@@ -45,6 +45,49 @@
         app:iconGravity="end"
         app:iconSize="28dp"
         app:iconTint="@color/iconColor"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0"
+        app:layout_constraintStart_toStartOf="parent"
         tools:text="Drive de test dev infomaniak" />
 
-</LinearLayout>
+    <!--  To prevent the button from overlapping the toolbar search button in the RootFilesFragment we do all of this complex
+    computations with views -->
```

Comment:
There is a `marginEnd` or something similar missing somewhere.

Reply:
The fact that the ripple stops before the end of the button in extrem cases is actually the intended behavior of this PR